app/assets/v2/js/notifications.js

```diff
@@ -0,0 +1,269 @@
+let notifications = [];
+let page = 1;
+let unreadNotifications = [];
+let hasNext = false;
+let numPages = '';
+let numNotifications = '';
+
+Vue.mixin({
```

Comment:
excited to learn more about Vue  -- itd becool to have a brownbag from you hearing how we can leverage Vue more for our codebase! 

cc @SaptakS

Reply:
I believe that explorer and bounty details are great candidates to be in Vue, will facilitate a lot the states changes from the json request.